test/library/standard/Sort/correctness/stableSort.chpl

```diff
@@ -0,0 +1,74 @@
+
+use Sort;
+
+record myRec {
+  var x: int;
+  var creationIndex: int;
+}
+record myRecComparator {
+  proc key(rec: myRec): int do
+    return rec.x;
+}
+proc factory(x: int): myRec {
+  @functionStatic
+  ref next = 0;
+  next += 1;
+  return new myRec(x, next);
+}
+
+record myRecComparatorByCreationIndex {
+  proc key(rec: myRec): int do
+    return rec.creationIndex;
+}
+proc inCreationOrder(arr) {
+  var inOrder = true;
+  forall (a, aprev) in zip(arr[arr.domain.low+1..],
+                           arr[arr.domain.low..arr.domain.high-1])
+    with (&& reduce inOrder) {
+    if aprev.x == a.x then
+      inOrder reduce= aprev.creationIndex < a.creationIndex;
+  }
+  return inOrder;
+}
+
+config const n = 1000;
+config const duplicates = 10;
+var arr: [0..#(n*duplicates)] myRec;
+for i in arr.domain by duplicates {
+  for j in 0..<duplicates {
+    arr[i+j] = factory(arr.domain.high - i);
+  }
+}
+
+config const printArray = false;
```

Comment:
It looks like you never set this to true via the command line.  Should this and the `print` function/calls just be removed?

Reply:
This was my debugging code while I was writing the test. I left it so that it exists in the future if someone ever needs to debug it (similar to how a ton of tests have `config const verbose = false` that never gets set to true). However I don't really have a strong preference on keeping it and can remove it if you prefer

Reply:
I don't feel strongly, it's fine to leave it